pysat/_files.py

```diff
@@ -28,12 +28,18 @@ class Files(object):
     ----------
     inst : pysat.Instrument
         Instrument object
+    data_dir : str or NoneType
+        Directory without sub-directory variables that allows one to
+        bypass the directories provided by pysat.params['data_dir'].  Only
+        applied if the directory exists. (default=None)
     directory_format : str or NoneType
-        Directory naming structure in string format. Variables such as
-        platform, name, tag, and inst_id will be filled in as needed using
-        python string formatting. The default directory structure would be
-        expressed as '{platform}/{name}/{tag}/{inst_id}'. If None, the default
-        directory structure is used (default=None)
+        Sub-directory naming structure, which is expected to exist within one
```

Comment:
It is absolutely true that pysat expects those directories to exist but pysat also makes them when downloading data. I wasn't able to think of a simple way to stick that in there. If there is a way though I think it would be helpful.